src/main/java/com/ldtteam/structurize/blocks/decorative/BlockTimberFrame.java

```diff
@@ -100,7 +98,7 @@ public TimberFrameFrameType getFrameType()
      *
      * @return the registered TimberFrameCentreType
      */
-    public TimberFrameCentreType getCentreType()
+    public TimberFrameCenterType getCenterType()
```

Comment:
it's spelt Centre outside the US: https://dictionary.cambridge.org/dictionary/english/centre

Reply:
I know! I *am* outside the U.S! But unfortunately they bet us to mass collaborative programming, and now they inflict their spelling conventions on us all. It was a toss-up, but I decided I would rather look at conventional spelling than proper spelling.

Reply:
it was fine as is, just don't change it lol